core/src/main/java/org/eclipse/hono/util/RequestResponseApiConstants.java

```diff
@@ -54,7 +54,11 @@
      * The name of the property that contains the identifier of a tenant.
      */
     public static final String FIELD_PAYLOAD_TENANT_ID = Constants.JSON_FIELD_TENANT_ID;
-
+    /**
+     * The name of the property that contains the boolean value defining whether
+     * one or more gateways may act on behalf of the device.
+     */
+    public static final String FIELD_PAYLOAD_GATEWAY_SUPPORTED = "gw-supported";
```

Comment:
this looks like it would better be placed into `RegistrationConstants` since it is very specific to the Device Registration API, isn't it?

Reply:
Right, I've changed that.